pinot-common/pom.xml

```diff
@@ -281,6 +282,11 @@
       <groupId>io.grpc</groupId>
       <artifactId>grpc-stub</artifactId>
     </dependency>
+    <dependency>
```

Comment:
I think versioned dependencies should go to root pom, here you just need to refer to that.

Reply:
Fixed.